plugins/foreman_discovery/nightly/index.md

```diff
@@ -223,7 +223,9 @@ on Fedora and Red Hat systems or
 
 on Debian and Ubuntu systems.
 
-Restart of Foreman is needed when installing the package separately.
+Restart of Foreman is needed when not using the installer:
+
+    # systemctl restart httpd
```

Comment:
On Debian/Ubuntu the service is called `apache2`

Reply:
Oh right thanks! :-)